lib/pbench/agent/tool_data_sink.py

```diff
@@ -169,7 +172,12 @@ def __init__(
         self.tool_group_dir = self.benchmark_run_dir / f"tools-{self.tool_group}"
 
     def launch(self):
-        pass
+        if not find_executable("podman"):
```

Comment:
This is debatable, but let's consider making this a base requirement of the Tool Meisters and Tool Data Sink, and determine this when they start up.